en/editor/publish/custom-build-plugin.md

```diff
@@ -44,92 +54,124 @@ export const configs: IConfigs = {
-    verifyRuleMap?: IVerificationRuleMap; // 注册参数校验规则函数
+    hooks?: string; // Storage path of hook function
+    options?: IDisplayOptions; // Platform parameter configuration that needs to be injected
+    verifyRuleMap?: IVerificationRuleMap; // Register parameter verification rule function
 }
 declare type IDisplayOptions = Record<string, IConfigItem>;
 declare interface IConfigItem {
-    // 默认值，注册的默认值将会在插件自身配置里的 options.[platform].xxx 字段内
+    // The default value, the registered default value will be in the options.[platform].xxx field in the plugin configuration
     default?: any;
 
     render: ?{
-        // 渲染 ui 组件规则，与 ui-prop 处统一规则一致，只有指定了 ui 属性的配置才会在构建配置面板上显示
+        // The rules for rendering ui components are consistent with the unified rules at ui-prop. Only configurations with ui properties specified will be displayed on the build configuration panel
         ui?: string;
-        // 传给 ui 组件的配置参数
+        // The configuration parameters passed to the ui component
         attributes?: IUiOptions;
     };
 
-    // 配置显示的名字，如果需要翻译，则传入 i18n:${key}
+    // Configure the displayed name, if you need to translate, then pass in i18n:${key}
     label?: string;
 
-    // 设置的简单说明，将会显示在配置名称的鼠标上移 title 上
+    // A brief description of the setting will be displayed on the title of the configuration name
     description?: string;
 
-    // 配置的类型
+    // Type of configuration
     type?: 'array' | 'object';
 
-    // 如果 type 是 array，则会按照指定数据类型和 itemConfigs 来渲染数据
+    // If type is an array, the data will be rendered according to the specified data type and itemConfigs
     itemConfigs?: Record<string, IConfigItem> | IConfigItem[];
 }
 
 declare interface IUiOptions extends IOptionsBase {
-    // 校验规则数组，构建提供一些基础规则，也可以通过 verifyRuleMap 来指定新的校验规则，只有当传入 require 时才会做无值的校验，否则仅存在值时才校验
+    // Validation rule array, build to provide some basic rules, you can also specify a new validation rule through verifyRuleMap, only when pass in require will be a valueless check, otherwise only when there is a value
     verifyRules?: string[];
 }
 
 declare interface IUiOptions extends IOptionsBase {
-    class?: string | string[]; // 需要设置在当前 ui-prop 上的样式名称
+    class?: string | string[]; // The name of the style that needs to be set on the current ui-prop
 }
 ```
 
-其中 `IOptionsBase` 的接口定义需要参考 [ui-prop 自动渲染规则定义](../extension/ui.md)。
+For the interface definition of `IOptionsBase` please refer to [ui-prop automatic rendering rule definition](../extension/ui.md).
+
+## Custom build hook function code configuration
 
-## 自定义构建钩子函数代码配置
+In the script module defined by the hooks field in the entry configuration, hook functionss can be written that build the life cycle. In different hook functions, the data received will be different. All hook functions run in the build process, and the engine method can be used directly in the build process. If you need to use `Editor`, adding the code `import * as Editor from'editor';` manually is required.
```

Comment:
1. functionss -> functions
2. adding the code `import * as Editor from 'editor';` to manually require.